.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+[*.{js,ts,vue}]
```

Comment:
I'd say we'd probably want to add html to that as well, as we will have at least one html to actually test the page.

Reply:
Possibly JSON too? (Though the current `package.json` is indented with two spaces.)

Reply:
Done.

Reply:
@lucaswerkmeister Sirry, didn't see your comment earlier. It is a good point and we probably want that for `.yml` as well. I'll address that in an extra PR that also fixes the two files